packages/ui/src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+
+import { HStack, VStack } from '../Stack';
+import { Checkbox, CheckboxProps } from '.';
+
+const UnCheckedCb = (props: Omit<CheckboxProps, 'onChange' | 'checked'>) => (
+  <Checkbox {...props} onChange={() => {}} checked={false} />
+);
+const CheckedCb = (props: Omit<CheckboxProps, 'onChange' | 'checked'>) => (
+  <Checkbox {...props} onChange={() => {}} checked />
+);
```

Comment:
I would consider if it's necessary to create such components. The only thing that changes is the `checked` prop and it would be shorter to just pass this property in particular stories (checkbox is unchecked by default)